.infrastructure/98_variables.tf

```diff
@@ -24,4 +24,19 @@ variable "tags" {
 ### required for security group ALB, ECS and RDS
 variable "cms_app_port" {
   default = 1337
+}
+
+variable "fargate_cpu" {
```

Comment:
```suggestion
variable "cms_app_cpu" {
```

Reply:
ok, I'll make the change with the next commit